src/propTypes/HandlePropTypes.js

```diff
@@ -0,0 +1,36 @@
+import PropTypes from 'prop-types';
+
+export default {
+  'aria-valuemax': PropTypes.number,
+  'aria-valuemin': PropTypes.number,
+  'aria-valuenow': PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+  'aria-disabled': PropTypes.bool,
+  'data-handle-key': PropTypes.node,
+  orientation: PropTypes.string,
+  disabled: PropTypes.bool,
+  onClick: PropTypes.func,
+  onKeyDown: PropTypes.func,
+  onMouseDown: PropTypes.func,
+  onTouchStart: PropTypes.func,
+  handleRef: PropTypes.func,
+  role: PropTypes.string,
+  style: PropTypes.object,
```

Comment:
this should come from withStylesPropTypes, instead of being defined in this repo

Reply:
this is `style` not `styles` (unfortunately).

Reply:
This is how it is currently in the repo
https://github.com/airbnb/rheostat/blob/master/src/Slider.jsx#L728

it's how Rheostat sets the handle position accurately